packages/documentation/src/pages/getting-started/common-tasks/run-build.mdx

```diff
@@ -0,0 +1,58 @@
+# Run and build VA.gov
+
+## Prerequisite
+
+- Complete the [first](/getting-started) step in *Start up guide*.
+
+## Locally run VA.gov
+
+Start the watch task:
+```bash
+yarn watch
+```
+### Build
+  - **Metalsmith** manages this build pipeline. At a high level, this process:
+    1. retrieves and **builds** all of the **static pages** sourced from `vagov-content` or Drupal
+    2. starts **Webpack**'s devevelopment server which
+       - **builds** the **JavaScript** and **CSS** bundles, and
+       - **serves** the **JavaScript** and **CSS** bundles and the **static content** built by Metalsmith at [http://localhost:3000](http://localhost:3000)
+
+### Output
+  - **Metalsmith** outputs **static pages** to`build/localhost`
+  - **Webpack** development server has **no output**. **JavaScript** and **CSS** are kept in memory.
+
+### Watch and rebuild
+  - **Metalsmith** will rebuild **static pages** when changes are saved to **templates** or content in `vagov-content`
+  - **Webpack** development server will rebuild when changes are saved to **JavaScript** or **SCSS**
+  - Both updates require a browser refresh to see changes.
+  - Changes to build process or configuration files are typically not picked up, and require a restart of the watch task to become active.
+
+
+## Locally build and run for specific environment
+
+_Note: most of the time, it's better to use the `watch` task to build the site locally since it is the most developer-friendly experience._
+
+| Environment | Build Command | Output directory | Run Command |
```

Comment:
![Screen Shot 2019-04-18 at 16 12 05](https://user-images.githubusercontent.com/4998130/56394467-ba7a4300-61f4-11e9-9d31-a6a33190ead7.png)

Reply:
tables are really important so i included screenshots to make them easier to review.